components/counter-bar/src/index.js

```diff
@@ -6,24 +6,23 @@ import ResultCountTitle from '@govuk-frederic/result-count-title';
 
 import { BLACK, GREY_1, GREY_3, LINK_COLOUR, RED, WHITE, YELLOW } from 'govuk-colours';
 
-const Container = styled('div')({
+const Wrapper = styled('div')({
   display: 'flex',
+  flexWrap: 'nowrap',
   lineHeight: '1',
 });
 
-const CountWrapper = styled('div')(
+const TotalWrapper = styled('a')(
   {
     ':focus': {
       outline: `solid 4px ${YELLOW}`,
     },
-    alignItems: 'center',
     border: '0',
-    display: 'inline-flex',
-    flex: '1',
-    marginRight: '8px',
+    flex: '0 0 auto',
```

Comment:
i'm not sure why this has changed

in normal use when we're putting a total next to several counters we'd want them all to flex, so that they all end up being of even size.  this change would make the total fixed width, based on the size of the enclosed item

Reply:
Right - can tweak that back again - I thought total was supposed to remain fixed width. You can still apply your class to it, I guess, setting a max/min width.

Reply:
for the most part, the `total` is intended to look/work the same as other items.  we have one exceptional case where, in use, it wants/needs to be fixed width

Reply:
As per Slack, it's actually proving rather tricky to have all boxes flex proportionately and yet have the items remain in their container and drop to the right of the total. I may need to just deprioritise this for the time being and get the other changes in...